tools/meson-git-contrib.sh

```diff
@@ -4,4 +4,5 @@ git shortlog -s `git describe --abbrev=0`.. | \
         cut -c8- | \
         sed 's/ / /g' | \
         awk '{ print $$0 "," }' | \
+        sed -e 's/ / /g' | \
```

Comment:
So... I'd prefer to drop this commit, but if you the contrib list looks with it, I'm OK with that.